### DISCUSSION TITLE
Arch Linux (and Manjaro etc) adaptation of Ubuntu-Server-ZFSbootmenu scripts based on Kayvlim etc

### DISCUSSION THREAD

**Discussion by Kodiman** [Sithuk/ubuntu-server-zfsbootmenu#26] [General]
Hi, Everyone, Thanks for the excellent Ubuntu-Server-ZFSbootmenu scripts. Suggest adapting these Ubuntu scripts to Arch Linux (and Manjaro etc). Perhaps, forking and modifying the Sithuk scrips with help from the Kayvlim examples. Is there anyone in the Arch Linux community who knows how to do this? Would this benefit the community to keep up with Ubuntu? 

https://github.com/Sithuk/ubuntu-server-zfsbootmenu

See this link for information and syntax about Arch Linux root install on ZFS...
https://wiki.archlinux.org/title/User:Kayvlim/Install_UEFI_and_BIOS_compatible_Arch_Linux_with_Encrypted_ZFS_and_ZFSBootMenu#Swap...

For reference, another effort at root on ZFS install scripts that is excellent, although not as developed as the Ubuntu scripts...
https://github.com/stevleibelt/arch-linux-configuration

Replies are welcome. Thanks

**Comment by Sithuk** [comment]
I'm not minded to create an arch zfsbootmenu version of the script at the moment. One advantage that an ubuntu based zfsbootmenu system has over arch is that Canonical releases its kernels with integrated zfs support so there is less of a risk that a kernel update will break the system.

**Comment by Anonymo** [comment]
> Hi, Everyone, Thanks for the excellent Ubuntu-Server-ZFSbootmenu scripts. Suggest adapting these Ubuntu scripts to Arch Linux (and Manjaro etc). Perhaps, forking and modifying the Sithuk scrips with help from the Kayvlim examples. Is there anyone in the Arch Linux community who knows how to do this? Would this benefit the community to keep up with Ubuntu?
> 
> https://github.com/Sithuk/ubuntu-server-zfsbootmenu
> 
> See this link for information and syntax about Arch Linux root install on ZFS... https://wiki.archlinux.org/title/User:Kayvlim/Install_UEFI_and_BIOS_compatible_Arch_Linux_with_Encrypted_ZFS_and_ZFSBootMenu#Swap...
> 
> For reference, another effort at root on ZFS install scripts that is excellent, although not as developed as the Ubuntu scripts... https://github.com/stevleibelt/arch-linux-configuration
> 
> Replies are welcome. Thanks

Yeah, for stevleibelt/arch-linux-configuration, I'm getting errors every time trying to get this installed.  Not sure why.
